### DISCUSSION TITLE
usb ip control question

### DISCUSSION THREAD

**Discussion by DDANGEUN** [gopro/OpenGoPro#282] [General]
hi. I have some question with gopro usb ip address.

use 172.2X.1YZ.51:8080 to communicate http.

and I believe 172.2X.1YZ.51 is only way to communicate with gopro.

But After many difficulties, I relized that I could access the udp stream at 172.2X.1YZ.5"2":8554.
It is also likely that many people like me(Beginer) do not know. documents only says "udp://@0.0.0.0:8554".
Of course, I was able to access the ipconfig and check that gopro ip address was 172.2X.1YZ.5"2".

By the way, What I'm curious about is the difference between the last digits 51 and 52.

This is a question for me that took a lot of time because I didn't have basic knowledge.
 
I'd appreciate it if you could answer. Thank you

**Comment by tcamise-gpsw** [comment]
Hi. I'm not sure exactly what you're asking for here. Is the problem discovering the GoPro USB IP address? Is the [example ](https://gopro.github.io/OpenGoPro/http_2_0#usb-2) not clear?

> For example, if the camera's serial number is C0000123456789, the IP address for USB connections would be 172.27.189.51.

**Reply by DDANGEUN** [reply]
I'm just curious about the meaning of the last letter of the IP address.
`cmd>ipconfig` I found my gopro ip4 address is 172.20.123.5**2**
and I can see udp stream through udp://@172.20.123.5**2**:8554
but send http command through 172.20.123.5**1**:8080
